trimmerdecorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0...10]
```

Comment:
- [ ] Here we have to trim the name when the `correct_name` is greater than 10 letters, I  suggest using this here:-
```
@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```

Reply:
thanks for the review.
it has been updated